src/Grok.Net/Grok.cs

```diff
@@ -86,6 +86,41 @@ public GrokResult Parse(string text)
             return new GrokResult(grokItems);
         }
 
+        public Dictionary<string, List<object>> AsDictionary(string text)
```

Comment:
This "additional" representation of the results is definitely NOT the responsibility of this class.
What if tomorrow we need another view? This approach violates the S and O from SOLID.

Reply:
Omg you're right I misread the issue.